cli.js

```diff
@@ -229,11 +259,14 @@ async function withdraw({ deposit, currency, amount, recipient, relayerURL, refu
     if (relayerURL.endsWith('.eth')) {
       throw new Error('ENS name resolving is not supported. Please provide DNS name of the relayer. See instuctions in README.md')
     }
-    const relayerStatus = await axios.get(relayerURL + '/status')
+    if (torPort) {
+      options = { httpsAgent: new SocksProxyAgent('socks://127.0.0.1:'+torPort), headers: { 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; rv:91.0) Gecko/20100101 Firefox/91.0' } }
```

Comment:
This will hide User-Agent exposing the version of node.js and os to protect privacy when `torPort` is enabled. `Mozilla/5.0 (Windows NT 10.0; rv:91.0) Gecko/20100101 Firefox/91.0` is the User-Agent of the latest version of Tor Browser.